test/github-issues/4646/entity/User.ts

```diff
@@ -0,0 +1,27 @@
+import {
+    BaseEntity,
+    Column,
+    Entity,
+    OneToMany,
+    PrimaryGeneratedColumn,
+} from "../../../../src"
+import { Photo } from "./Photo"
+
+@Entity({
+    versioning: {
+        validFrom: "valid_from",
+        validTo: "valid_to",
+        dataConsistencyCheck: false,
+        historyTable: "user_history",
+    },
+})
```

Comment:
what if I had "versioning" and now I removed it from my entity? Will synchronization work properly? (e.g. remove everything related to versioning and keep my data not related to versioning untouched)

Reply:
I'll check it an add a unit test for that case.

Reply:
I implemented the method `changeTableVersioning` to the QueryRunner and add a unit test.